### PR TITLE
a simple bootstrap integration on a basic webpage

### PR DESCRIPTION
**Pull request by JeanGarcia**
Here's a basic webpage integrated with Bootstrap 4 to make it responsive. It has examples of the basic components personalized by me.

### CODE REVIEW

**Comment by JeanGarcia** [comment]
example

**Review by MallikarjunHt — APPROVED** [review]

**Comment by JeanGarcia** [comment]
Hi, could this PR be accepted/merged? I saw that it was already approved but i think it won't be reviewed until is merged or accepted with the Hactoberfest tag. Sorry for rushing this up, but it takes some time for the PRs to be reviewed by the maintainers of the event so the sooner it enters on their queue the more possibilities are that the PR is reviewed on time before the event ends.

**Pull request merged**
